SSW.Rules.AzFuncs/Functions/Widget/GetLatestRules.cs

```diff
@@ -31,6 +31,7 @@ public class GetLatestRules(ILoggerFactory loggerFactory, RulesDbContext context
                         r.CreatedBy == githubUsername || r.UpdatedBy == githubUsername)
             .GroupBy(r => r.RuleGuid)
             .Select(group => group.First())
+            .Where(r => r.ArchivedReason is not null)
```

Comment:
wait this should be null right? 

i.e. we don't want archived rules showing up

Reply:
Yep! you're right, fixed.